### PR TITLE
Simplify rolling tips and tricks

### PR DESCRIPTION
**Pull request by jsignell**
Let me know what you think. Or if you want it evaluated.

### CODE REVIEW

File: tips-and-tricks/rolling_averages_with_dask.ipynb [hunk 1]
```diff
@@ -224,262 +148,35 @@
...
-    "_ = wait(fares)"
+    "taxi = taxi.set_index(\"tpep_pickup_datetime\")"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Let's take a quick look and see what we're working with."
+    "We can check the boundaries of our partitions by calling `taxi.divisions`. Note that the paritions aren't evenly distributed across time, but this doesn't matter. Let's look at 10 just to get a sense."
```
Comment (skirmer) [inline tips-and-tricks/rolling_averages_with_dask.ipynb:170]: "partitions"

**Review by skirmer — CHANGES_REQUESTED** [review]
Just a typo, otherwise I think it's great!

File: tips-and-tricks/rolling_averages_with_dask.ipynb (hunk 1 above)
Comment (jsignell) [reply tips-and-tricks/rolling_averages_with_dask.ipynb:170]:
```suggestion
    "We can check the boundaries of our partitions by calling `taxi.divisions`. Note that the partitions aren't evenly distributed across time, but this doesn't matter. Let's look at 10 just to get a sense."
```

**Comment by jsignell** [comment]
I'm pushing the evaluated notebook instead in 2 min.

**Comment by jsignell** [comment]
Ok @skirmer will merge when you :heavy_check_mark:

**Review by skirmer — APPROVED** [review]
🎉

**Pull request merged**